.github/workflows/release-trustframeworkadoption.yml

```diff
@@ -0,0 +1,77 @@
+name: Create TrustFrameworkAdoption Release
+on:
+  workflow_dispatch:
+    inputs:
+      tfa_version:
+        description: 'Version string that is used for publishing (e.g. "1.0.0", NOT "v1.0.0"). Appending -SNAPSHOT will create a snapshot release.'
+        required: true
+        type: string
+
+
+env:
+  TRUSTFRAMEWORKADOPTION_VERSION: ${{ github.event.inputs.tfa_version || inputs.tfa_version }}
+
+jobs:
+  Prepare-Release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      # create tag on the current branch using GitHub's own API
+      - name: Create tag on current branch (main)
+        uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: 'refs/tags/v${{ env.TRUSTFRAMEWORKADOPTION_VERSION }}',
+              sha: context.sha
+            })
+
+      # create merge commit main -> releases encoding the version in the commit message
+      - name: Merge main -> releases
+        uses: everlytic/branch-merge@1.1.4
+        with:
+          github_token: ${{ github.token }}
+          source_ref: ${{ github.ref }}
+          target_branch: 'releases'
+          commit_message_template: 'Merge commit for release of version v${{ env.TRUSTFRAMEWORKADOPTION_VERSION }}'
+
+      # Trigger EF Jenkins. This job waits for Jenkins to complete the publishing, which may take a long time, because every
+      # module is signed individually, and parallelism is not available. Hence, the increased timeout of 3600 seconds.
+      # There is no way to cancel the process on Jenkins from withing GitHub.
+      - name: Trigger Release on EF Jenkins
+        uses: toptal/jenkins-job-trigger-action@master
+        with:
+          jenkins_url: "https://ci.eclipse.org/dataspaceconnector/"
+          jenkins_user: ${{ secrets.EF_JENKINS_USER }}
+          jenkins_token: ${{ secrets.EF_JENKINS_TOKEN }}
+          job_name: "TrustFrameworkAdoption-Autobuild-Release"
```

Comment:
I doubt that this Jenkins job exists :)
If TrustFrameworkAdoption should be published alongside all other EDC artefacts, we probably wanna include it in the [Releases Repo](https://github.com/eclipse-edc/Release) and modify the [Publish-All-in-One](https://ci.eclipse.org/edc/job/Publish-All-In-One/) job. For now, I'd just remove the `Trigger Release on EF Jenkins` step, if not the entire workflow.

Reply:
We actually want this repo to publish artifacts, but let's treat the releasing part in a separated PR. 

I removed the workflow for now.